raiden/tests/unit/transfer/mediated_transfer/test_initiatorstate.py

```diff
@@ -176,17 +185,24 @@ def test_next_route():
 
 
 def test_init_with_usable_routes():
+    transfer_amount = TokenAmount(1000)
+    flat_fee = FeeAmount(20)
+    expected_fee_margin = 1  # 5% of 20
```

Comment:
Whenever we change `DEFAULT_MEDIATION_FEE_MARGIN` this will break. Perhaps remove the comment and instead calculate it directly here as `DEFAULT_MEDIATION_FEE_MARGIN * flat_fee` ?